Include/pymath.h

```diff
@@ -56,24 +56,14 @@
  * doesn't support NaNs.
```

Comment:
With this change, the comment about `INF*0` or `INF/INF` working is out of date; that sentence should probably be removed.